internal/pkg/model/state.go

```diff
@@ -223,44 +226,6 @@ func (s *State) GetOrCreateFrom(manifest ObjectManifest) (ObjectState, error) {
 	return s.CreateFrom(manifest)
 }
 
-func (s *State) IsTracked(path string) bool {
-	return s.pathsState.IsTracked(path)
-}
-
-func (s *State) IsUntracked(path string) bool {
-	return s.pathsState.IsUntracked(path)
-}
-
-// TrackedPaths returns all tracked paths.
-func (s *State) TrackedPaths() []string {
-	return s.pathsState.TrackedPaths()
-}
-
-// UntrackedPaths returns all untracked paths.
-func (s *State) UntrackedPaths() []string {
-	return s.pathsState.UntrackedPaths()
-}
-
-func (s *State) UntrackedDirs() (dirs []string) {
-	return s.pathsState.UntrackedPaths()
-}
-
-func (s *State) UntrackedDirsFrom(base string) (dirs []string) {
-	return s.pathsState.UntrackedDirsFrom(base)
-}
-
-func (s *State) IsFile(path string) bool {
-	return s.pathsState.IsFile(path)
-}
-
-func (s *State) IsDir(path string) bool {
-	return s.pathsState.IsDir(path)
-}
-
-func (s *State) LogUntrackedPaths(logger *zap.SugaredLogger) {
-	s.pathsState.LogUntrackedPaths(logger)
-}
-
```

Comment:
Tieto metody su vyriesene pomocou embeddingu, vid vyssie `pathsRO`.
Je to tak aspon trocha prehladnejsie.